xla/service/gpu/ir_emitter_triton.cc

```diff
@@ -148,6 +149,15 @@ using ::mlir::Type;
 using ::mlir::Value;
 using mlir::ValueRange;
 
+
+std::string GetLibdevicePath(const HloModuleConfig& hlo_config,
+                             const se::DeviceDescription& device_info);
+
+absl::Status CreateTritonPipeline(
+    mlir::OpPassManager& pm, const se::GpuComputeCapability& cc,
+    const TritonGemmConfig& config,
+    mt::nvidia_gpu::ClusterInfo& out_cluster_info);
+
```

Comment:
It's an interesting choice that these are not in the header.
I mean I get the point that these are kindof private to ir_emitter_triton, but still, we usually put function declarations in headers.
If it's not too difficult, please try to do that.

Reply:
OK, I will made changes that you have suggested.